spec/unit/api_client_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'spec_helper'
+
+describe Typekitter::ApiClient do
+  describe '#kits_index' do
+    context 'request to typekit api is successful' do
+      it 'returns a parsed string of typekit kits' do
+        json_response = "{\"foo\":[\"success\"]}"
+        parsed_kits = JSON.parse(json_response)
+        stub_request(:get, "https://typekit.com/api/v1/json/kits").
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.